distribution/resources/bin/gateway

```diff
@@ -45,11 +45,13 @@ PRGDIR=`dirname "$PRG"`
 
 # set BALLERINA_HOME
 GW_HOME=`cd "$PRGDIR/.." ; pwd`
-export JAVA_HOME=$GW_HOME'/lib/jdk8u202-b08-jre'
 export MGW_VERSION="3.1.0"
 
 JAVA_PATH=$GW_HOME/lib/jdk8u202-b08-jre
-if [ -d "$JAVA_PATH" ]; then
+if [ -d "$JAVA_HOME" ]; then
+  echo "JAVA_HOME is externally set: $JAVA_HOME"
+else
+  echo "JAVA_HOME..."
```

Comment:
Remove this
```suggestion
```

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/966/commits/fe1f68c2c9d1d39e961d2664256249498d69c5aa